src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -65,10 +80,8 @@ private ModBlocks()
     /**
      * Initates all the blocks. At the correct time.
      */
-    public static void init(final IForgeRegistry<Block> registry)
-    {
+    public static void init(final IForgeRegistry<Block> registry) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)